components/base/kinematicbase/fake_kinematics.go

```diff
@@ -84,3 +87,46 @@ func (fk *fakeKinematics) GoToInputs(ctx context.Context, inputs []referencefram
 	time.Sleep(150 * time.Millisecond)
 	return nil
 }
+
+//nolint: dupl
+func (fk *fakeKinematics) ErrorState(
```

Comment:
I'm not sure what the point of doing all this is when there will never be any errors in the fake model to begin with.  In the future we might want to think about spoofing some error, but for now I think this can effectively return a zero Pose